README.md

```diff
@@ -0,0 +1,113 @@
+# pmemfile
+
+Pmemfile project's goal is to provide low-overhead userspace implementation of
+file APIs using persistent memory.
+It consists of 2 major compoments:
+- libpmemfile-posix - provides syscall-like API which can be used by applications
+- libpmemfile - provides transparent access to libpmemfile-posix pools
+
+# How to build #
+
+Requirements:
+- cmake >= 3.3 (XXX: can we build with older versions?)
```

Comment:
The oldest version I tested pmemfile with was 3.3, but if we are not using features introduced in this version we could users build it with older versions.